### PR TITLE
Hasan/Pr accumulator why trade

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/components/molecules/trade-item/style.module.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 .trade_item {
     @include breakpoints(tablet){
-        max-inline-size: 33%;
+        max-inline-size: 237px;
```
Comment (mayuran-deriv) [inline src/features/components/molecules/trade-item/style.module.scss]: possible to use rem ?

File: src/features/components/templates/why-trade-options/index.tsx [hunk 2]
```diff
@@ -43,14 +45,20 @@ const WhyTradeOptions = ({ data }: { data: WhyTradeOptionsType }) => {
                     ))}
                 </Flex.Box>
                 <Flex.Box direction="col" align="center" mt="12x" md={{ mt: '20x' }}>
-                    <Typography.Heading as="h4" textcolor="inverted" size="xxs" mb="8x">
+                    <Typography.Heading
+                        as="h5"
+                        textcolor="inverted"
+                        size="xxs"
+                        mb="8x"
+                        className={dclsx('text-medium')}
```
Comment (mayuran-deriv) [inline src/features/components/templates/why-trade-options/index.tsx:53]: do we need size and  text medium ?

Comment (hasan-deriv) [reply src/features/components/templates/why-trade-options/index.tsx:53]: To align with design.

File: src/features/components/molecules/trade-item/style.module.scss (hunk 1 above)
Comment (hasan-deriv) [reply src/features/components/molecules/trade-item/style.module.scss]: Yeah, doing it...

**Review by aswathy-deriv — APPROVED** [review]

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**